Fix installing Amazon EC2 plugin (#4190)
theforeman/foreman-documentation: guides/common/modules/proc_installing-amazon-ec2-plugin.adoc

This allows you to manage and deploy hosts to EC2.

.Procedure
* Install the EC2 compute resource provider on your {ProjectServer}:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# {foreman-installer} --enable-foreman-compute-ec2
----

.Verification

> * In the {ProjectWebUI}, navigate to *Administer* > *About* and select the *Available Providers* tab to verify the installation of the Amazon EC2 plugin.

Reviewer: ```suggestion
* In the {ProjectWebUI}, navigate to *Administer* > *About*.
* On the *Available Providers* tab, verify the installation of the Amazon EC2 plugin.
```

Author: ```suggestion
. In the {ProjectWebUI}, navigate to *Administer* > *About*.
. On the *Available Providers* tab, verify the installation of the Amazon EC2 plugin.
```
In that case we have to use a numbered list.

Author: Only when it has a single step.
